rom/dev/tools/test-fmc/src/main.rs

```diff
@@ -195,32 +195,44 @@ fn get_pcr_entry(entry_index: usize) -> PcrLogEntry {
     PcrLogEntry::read_from_prefix(pcr_entry.as_bytes()).unwrap()
 }
 
-fn process_mailbox_commands() {
-    let mut mbox = unsafe { caliptra_registers::mbox::MboxCsr::new() };
-    let mbox = mbox.regs_mut();
-
+fn process_mailbox_command(mbox: &caliptra_registers::mbox::RegisterBlock<RealMmioMut>) {
     let cmd = mbox.cmd().read();
     cprintln!("[fmc] Received command: 0x{:08X}", cmd);
     match cmd {
         0x1000_0000 => {
-            read_pcr_log(&mbox);
+            read_pcr_log(mbox);
         }
         0x1000_0001 => {
-            create_certs(&mbox);
+            create_certs(mbox);
         }
         0x1000_0002 => {
-            read_fuse_log(&mbox);
+            read_fuse_log(mbox);
         }
         0x1000_0003 => {
-            read_fht(&mbox);
+            read_fht(mbox);
         }
         0x1000_0004 => {
-            trigger_update_reset(&mbox);
+            trigger_update_reset(mbox);
         }
         _ => {}
     }
 }
 
+fn process_mailbox_commands() {
```

Comment:
Another option would be to run these tests from RT. Not sure if the tests you need to run have to be done in FMC or not.

Reply:
Yes, that's an option, would need to add handoff code to this FMC, since it doesn't know how to boot an RT image. I think the idea behind this test-fmc was to just be a minimal image, and while we package RT FW with it in tests, I don't think that RT fw ever actually runs.

But yeah we could just take all the host commands in this image and move them out to RT FW.

Reply:
The RT tests use the regular FMC instead of the test FMC. Would that be a problem?

I'm not super convinced test-fmc needs to exist at all now that real FMC exists (but would be fine leaving it for the tests that already depend on it).

Reply:
Yeah I think test-fmc and test-rt could go away.

However, that migration looks fairly involved. Mind if we just extend test-fmc in the short term to unblock test coverage for new PCR functionality?